kratos/includes/define.h

```diff
@@ -40,12 +40,14 @@
      #define KRATOS_ENV64BIT
    #else
      #define KRATOS_ENV32BIT
+     #error 32 bit system are not supported anymore. Please consider a 64 bits system
   #endif
 #else // It is POSIX (Linux, MacOSX, BSD...)
   #if defined(__x86_64__) || defined(__ppc64__) || defined(__aarch64__)
     #define KRATOS_ENV64BIT
   #else // This includes __arm__ and __x86__
     #define KRATOS_ENV32BIT
+     #error 32 bit system are not supported anymore. Please consider a 64 bits system
```

Comment:
Can this be checked with CMake?
Seems like the better place to do this

Reply:
We can...

Reply:
Do you know how to check that in CMake?, I can't find an easy manner

Reply:
I would just not check it, 32 bit users are scarse

Reply:
But without this those scarce developers will be able to compile, and they will have an execution error not explained when using a variable and the hash induced and overflow

Reply:
Maybe we can just inform about this in the wiki compilation guide.

Reply:
You know that our main problem is that people don't read the documentation, I think the check is relevant and must be done

Reply:
But still we have the possibility that someone directly compiles without reading it. It is safer this way

Reply:
Idk to be honest, I see @loumalouomega points, i don't like it but if you really want to leave it I have no problems with it. Maybe we can remove it later. What do you think?